### PR TITLE
Add interrupt pending

### PR DESCRIPTION
**Pull request by schwitzguebel**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Review by danielthirion — APPROVED** [review]

File: gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/configurations/UrlConfiguration.java [hunk 1]
```diff
@@ -19,10 +19,19 @@
...
 
     public List<String> getWhitelist() {
         return whitelist;
     }
+
+    public String getInterruptServerUrl() {
+        return interruptServerUrl;
+    }
+
+    public void setInterruptServerUrl(String interruptServerUrl) {
```
Comment (vbochetRTE) [inline gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/configurations/UrlConfiguration.java:34]: Please use a valued constructor instead

File: gridcapa-swe-runner-app/src/test/java/com/farao_community/farao/swe/runner/app/services/FileImporterTest.java [hunk 2]
```diff
@@ -78,14 +78,6 @@ void testImportCimCrac() {
 
     @Test
     void testImportCimCracFromUrlWithNetwork() {
-        Properties importParams = new Properties();
```
Comment (vbochetRTE) [inline gridcapa-swe-runner-app/src/test/java/com/farao_community/farao/swe/runner/app/services/FileImporterTest.java:81]: Why did you delete this?

**Review by vbochetRTE — APPROVED** [review]
